src/services/anchor-service.ts

```diff
@@ -488,11 +488,10 @@ export class AnchorService {
       for (const candidate of candidates) {
         groupedRequests.acceptedRequests.push(...candidate.acceptedRequests)
       }
-      logger.debug(
-        `Marking ${groupedRequests.acceptedRequests.length} pending requests as processing`
-      )
+
+      // set candidates that will be anchored as pinned to be garbage collected later
       await this.requestRepository.updateRequests(
```

Comment:
I think we should remove this update entirely.  It does mean that if we crash some streams may be pinned that aren't marked that way in the database, but that's already the case today, this just makes that window slightly larger.  But in the vast majority of cases, those streams will be picked up in the next batch and if that batch finishes successfully then the state will be correct.

This update slows things down and floods the logs, for very little real value.